src/modlunky2/ui/trackers/category.py

```diff
@@ -33,19 +34,24 @@ def __init__(self, parent, modlunky_config: Config, *args, **kwargs):
             Image.open(ICON_PATH / "cat2.png").resize((24, 24), Image.ANTIALIAS)
         )
 
+        current_row = 0
```

Comment:
I think the variable is approach is fine and used elsewhere.

As an alernative, you could make a list of widgets and then use `for row, widget in enumerate(some_widget_list): widget.grid(opts_here)`. The builtin `enumerate()` produces (index, value) pairs, i.e. you could use the index of the list item as its row number